pyvista/plotting/utilities/cubemap.py

```diff
@@ -0,0 +1,135 @@
+"""Cubemap utilities."""
```

Comment:
This is an oddly specific module. Was it so fine-grained before this refactor?

Reply:
I wasn't too sure where to put these. They were previously in `pyvista.utilities.helpers`... perhaps I could put them in `pyvista.plotting.utilities.helpers`? I made a dedicated module because there were at least 3 "cubemap" related things

Reply:
Well, they smell a bit too fine-grained for me. But this is not a very strong stance on my part. We can also leave as is.